pep-0370.txt

```diff
@@ -203,21 +201,21 @@ References
    http://peak.telecommunity.com/DevCenter/EasyInstall#creating-a-virtual-python
 
 .. [2]  Working Env
-   http://pypi.python.org/pypi/workingenv.py
-   http://blog.ianbicking.org/workingenv-revisited.html
+   https://pypi.org/project/workingenv.py/
+   https://ianbicking.org/archive/workingenv-revisited.html
 
 .. [3] Virtual Env
-   http://pypi.python.org/pypi/virtualenv
+   https://pypi.org/project/virtualenv/
 
 .. [4] reference implementation
    http://bugs.python.org/issue1799
    http://svn.python.org/view/sandbox/trunk/pep370
 
 .. [5] MSDN: CSIDL
-   http://msdn2.microsoft.com/en-us/library/bb762494.aspx
+   https://learn.microsoft.com/en/windows/win32/shell/csidl
 
-.. [6] Initial suggestion for a per user site-packages directory
-   http://permalink.gmane.org/gmane.comp.python.devel/90902
+[6] Initial suggestion for a per user site-packages directory
+\   http://permalink.gmane.org/gmane.comp.python.devel/90902
```

Comment:
The original is in the Wayback Machine: https://web.archive.org/web/20080511032648/http://permalink.gmane.org/gmane.comp.python.devel/90902 

So we can find its replacement:

```suggestion
\   https://mail.python.org/archives/list/python-dev@python.org/message/V23CUKRH3VCHFLV33ADMHJSM53STPA7I/
```

(I didn't check the others)

Reply:
Good idea!